src/App/Hovedside/TjenesteBoks/TjenesteBoks.tsx

```diff
@@ -13,20 +14,20 @@ interface Props {
 
 const TjenesteBoks: FunctionComponent<Props> = props => {
   return (
-    <div className={"tjenesteboks"}>
-      <img className={"tjenesteboks__icon"} src={props.bildeurl} />
-      <div className={"tjenesteboks__tekstboks"}>
-        <Undertittel className={"tjenesteboks__header"}>
-          {props.tittel}
-        </Undertittel>
-        <Normaltekst className={"tjenesteboks__undertekst"}>
-          {props.undertekst}
-        </Normaltekst>
-        <Lenke href={props.lenke}>
-          {props.lenketekst}
-        </Lenke>
+    <Innholdsboks>
+      <div className={"tjenesteboks"}>
```

Comment:
Vi kan flytte className propen til Innholdsboks slik at vi slipper å ha de ekstra divene bare for å slenge på en ekstra div inni hver Innholdsboks. Jeg kan vise hvordan man gjør det :)